_site/index.html

```diff
@@ -6,8 +6,9 @@
     <meta name="viewport" content="width=device-width, initial-scale=1">
     <link rel="shortcut icon" type="image/ico" href="/images/favicon.ico">
     <link rel="stylesheet" href="css/main.css">
+    <script> window.initMap = function () { } </script>
```

Comment:
Al cargar la api de google maps, es obligatorio pasar el nombre del callback a ejecutar, pero realmente no estamos usando uno, sino que todo lo hacemos desde `main.js`. Cuando el JS de google maps no encuentra la función, se produce este error:
`Uncaught (in promise) InvalidValueError: initMap is not a function`

La forma más sencilla de evitarlo es declarar la función aunque sea vacía, considero innecesario hacer un refactor del JS para que google maps llame al callback "real".

Reply:
Por qué no consideras necesario hacer el cambio? 🤔 

Creo que quedaría más limpio si asignamos `showGoogleMaps` a `window.initMap`, entiendo que es sencillo.